src/js/modules/breakpointHelper.js

```diff
@@ -60,3 +62,57 @@ export const getCurrentScreenWidth = () => {
     // If no breakpoints have been set
     return false;
 };
+
+export const withinBreakpoint = breakpointString => {
+    const operatorMatch = breakpointString.match(/[<>=]+/);
+    const operator = operatorMatch ? operatorMatch[0] : '';
+    const [, breakpoint] = breakpointString.split(/[<>=]+/);
+    const currentScreenWidth = window.innerWidth;
+    let match;
+
+    // We fire getCurrentScreenWidth to set the bp array we loop through to get the PX values
+    getCurrentScreenWidth();
+
+    // We loop through the breakpoint array until we get a match.
+    // If we match we return the px value as an int. If we do not match we return false
+    const breakpointToPX = bp => {
+        for (let i = 0; i < bps.length; i++) {
+            if (bps[i][0] === bp) {
+                return parseInt(bps[i][1], 10);
+            }
+        }
+        return false;
+    };
+
+    const breakpointInPX = breakpointToPX(breakpoint);
+
+    // If the breakpoint passed in does not match any we return false;
+    if (breakpointInPX === false) {
+        return false;
```

Comment:
Do you need to return false here? `breakpointToPX` already returns false you could just `return` instead?

Reply:
I get shouted at by the linter for inconsistent returns if i dont